awscli/examples/cloudwatch/start-metric-streams.rst

```diff
@@ -0,0 +1,8 @@
+**To start a specified metric stream**
+
+The following ``start-metric-streams`` example starts the metric stream named ``QuickFull-GuaFbs`` in the specified account. If the command succeeds, no output is returned. ::
```

Comment:
Same here. The line for "This command produced no output." should be moved to after the command example where the output section is.
https://aws.github.io/aws-cli/docs_styleguide.html#command-produces-no-output